libsolidity/codegen/MultiUseYulFunctionCollector.h

```diff
@@ -29,6 +29,16 @@
 namespace solidity::frontend
 {
 
+struct YulUtilFunction
+{
+	std::string name;
+	std::vector<std::string> parameters;
+	std::vector<std::string> returnValues;
+	std::string body;
+
+	std::string code() const;
```

Comment:
`code()` sounds like simple accessor so I think it might encourage people to just call it multiple times without caching the result in between.

```suggestion
	std::string render() const;
```